src/test/java/core/basesyntax/service/impl/WriterServiceImplTest.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.WriterService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class WriterServiceImplTest {
+    private static final String REPORT_FILEPATH = "src/test/resources/reportTest.csv";
+    private static WriterService writerService;
+    private static String report;
+
+    @Rule
+    public ExpectedException exceptionRule = ExpectedException.none();
+
+    @BeforeClass
+    public static void beforeClass() {
+        writerService = new WriterServiceImpl();
+        report = "fruit,quantity"
+                + System.lineSeparator()
+                + "banana,100"
+                + System.lineSeparator()
+                + "apple,100"
+                + System.lineSeparator();
+    }
+
+    @Test
+    public void writeTo_correctReport_ok() throws IOException {
```

Comment:
what if writing works correctly but we can't read correctly written data and get exception?